### DISCUSSION TITLE
Very important software Update

### DISCUSSION THREAD

**Discussion by ihattan2020** [neurodesk/neurodesk.github.io-hugo#450] [Q&A]
Dear NeuroDesk Team,

I would like to have the below software with up-to-date version. Could you please help to get these modules updated?

below is the link for latest versions for Spinalcordtoolbox (v. 6.4)and ANTs (v. 2.5.3)

-https://github.com/spinalcordtoolbox/spinalcordtoolbox/releases

-https://github.com/ANTsX/ANTs/releases

Many thanks in advance for your help.

Kind Regards,

Ibrahim

**Comment by stebo85** [comment]
Dear @ihattan2020,

yes - that's possible. Can you propose these changes directly through a pull request?

https://github.com/NeuroDesk/neurocontainers/blob/master/recipes/spinalcordtoolbox/build.sh

https://github.com/NeuroDesk/neurocontainers/blob/master/recipes/ants/build.sh
